README.md

```diff
@@ -106,281 +177,224 @@ yarn deploy-local
                 lastResultsHash
                 evidenceHash
                 proposerAddress
-              }
-              commit {
```

Comment:
Before removing this, is it being captured anywhere else ? Remember we want this query to output ALL data from the block.